awscli/examples/eks/wait/fargate-profile-deleted.rst

```diff
@@ -0,0 +1,9 @@
+**To wait for an fargate-profile running in the Amazon EKS cluster to become deleted**
+
+The following ``wait fargate-profile-deleted`` example command waits until `ResourceNotFoundException` is thrown when polling with `describe-fargate-profile` for an fargate-profile named ``my-fargate-profile`` running in the Amazon EKS cluster named ``my-eks-cluster``. ::
```

Comment:
`ResourceNotFoundException` and `describe-fargate-profile` need an extra set of backticks.